### PR TITLE
Allow watchtower to update rebooting containers

### LINKED ISSUE #650: Updating containers that is rebooting
Hi,
if we at some point we introduce a problem in a docker container that cause crashing of the container then it will be standing in a reboot loop if --restart always is set on that container. Even if you include WATCHTOWER_INCLUDE_STOPPED=true then it will not update the container when it standing in a reboot loop. I though it would be a good idea to introduce a new setting WATCHTOWER_INCLUDE_REBOOTING so that when the docker container get fixed then it will pull down the new one. Restoring the operation of the docker container. 

rebooting status is not included in filter:
https://github.com/containrrr/watchtower/blob/98b518612b6ae2c005667de997bec67325cd98cf/pkg/container/client.go#L102-L112

### PR DESCRIPTION
**Pull request by yrien30**
Closes #650

### CODE REVIEW

**Comment by yrien30** [comment]
Hi, was going to make a new pull request for #649. I wanted to have it in seperate pull request but since I did't start whit making a new branch before I started to make changes it came on same. I reverted the change for now. I will create new pull request for this when you have taken in the changes for #650.

**Comment by piksel** [comment]
I just removed the referenced ticket in the title, because it messes with Githubs autocomplete for references

File: cmd/root.go [hunk 1]
```diff
@@ -116,6 +116,7 @@ func PreRun(cmd *cobra.Command, args []string) {
 
 	noPull, _ := f.GetBool("no-pull")
 	includeStopped, _ := f.GetBool("include-stopped")
+	includeRestarting, _ := f.GetBool("includeRestarting")
```
Comment (piksel) [inline cmd/root.go]: Typo?
```suggestion
	includeRestarting, _ := f.GetBool("include-restarting")
```

**Review by piksel — CHANGES_REQUESTED** [review]
This overall looks good to me.

The flag name doesn't align with the current ones though.

File: pkg/container/container_test.go [hunk 2]
```diff
@@ -68,6 +69,18 @@ var _ = Describe("the container", func() {
...
+				client = dockerClient{
+					api:               docker,
+					pullImages:        false,
+					includeRestarting: true,
+				}
+				containers, err := client.ListContainers(filters.NoFilter)
+				Expect(err).NotTo(HaveOccurred())
+				Expect(len(containers) > 0).To(BeTrue())
+			})
+		})
```
Comment (simonaronsson) [inline pkg/container/container_test.go]: could you please add a test case for the opposite as well? not including restarted when the flag is not set, that is.

**Review by simonaronsson — CHANGES_REQUESTED** [review]

File: pkg/container/container_test.go (hunk 2 above)
Comment (yrien30) [reply pkg/container/container_test.go]: Sure 👍

**Review by simonaronsson — APPROVED** [review]

**Comment by yrien30** [comment]
Hi,  is there anything more to do on this or can it be merged now?

**Review by piksel — APPROVED** [review]

**Comment by simonaronsson** [comment]
Thank you kindly for your contribution! 🙇

**Comment by simonaronsson** [comment]
@all-contributors add @yrien30 for code

**Pull request merged**